src/SignalR/src/Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson/Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Implements the SignalR Hub Protocol over JSON.</Description>
+    <Description>Implements the SignalR Hub Protocol using JSON over Newtonsoft.Json.</Description>
```

Comment:
nit: "over Newtonsoft.Json" sounds weird. Maybe simplify to just "using Newtonsoft.Json"